USB/USBCDC_ECM/README.md

```diff
@@ -0,0 +1,11 @@
+## USBCDC_ECM example
```

Comment:
One the API documentation merges, I'll update this README to point to it in a new PR.